<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdImageHistogramGraphControls</class>
 <widget class="QWidget" name="QxrdImageHistogramGraphControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Image Histogram(s)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Full</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Hist A</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Hist B</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Hist C</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="m_HistFullVis">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="m_HistAVis">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="m_HistBVis">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="m_HistCVis">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="m_HistALeft"/>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="m_HistBLeft"/>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="m_HistCLeft"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Top</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="m_HistATop"/>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="m_HistBTop"/>
   </item>
   <item row="4" column="4">
    <widget class="QSpinBox" name="m_HistCTop"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="m_HistARight"/>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="m_HistBRight"/>
   </item>
   <item row="5" column="4">
    <widget class="QSpinBox" name="m_HistCRight"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Bottom</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="m_HistABottom"/>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="m_HistBBottom"/>
   </item>
   <item row="6" column="4">
    <widget class="QSpinBox" name="m_HistCBottom"/>
   </item>
   <item row="7" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>378</width>
       <height>306</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Visible?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
